scene/main/viewport.cpp

```diff
@@ -3242,6 +3242,33 @@ float Viewport::get_lod_threshold() const {
 	return lod_threshold;
 }
 
+void Viewport::set_use_occlusion_culling(bool p_use_occlusion_culling) {
+	if (use_occlusion_culling == p_use_occlusion_culling) {
+		return;
+	}
+
+	use_occlusion_culling = p_use_occlusion_culling;
+	RS::get_singleton()->viewport_set_use_occlusion_culling(viewport, p_use_occlusion_culling);
+
+	notify_property_list_changed();
+}
+
+bool Viewport::is_using_occlusion_culling() const {
+	return use_occlusion_culling;
+}
+
+void Viewport::set_occlusion_rays_per_thread(int p_rays_per_thread) {
```

Comment:
I think this should probably be a global setting for all viewports, since it does not make much sense to change. like `RenderingServer::viewport_set_occlusion_rays_per_thread(amount)` and it works for all viewports. Default value goes into a GLOBAL_DEF().